release-notes/6.0/preview/api-diff/preview1/.Net/6.0-preview1_System.Net.Http.md

```diff
@@ -0,0 +1,10 @@
+# System.Net.Http
+
+``` diff
+ namespace System.Net.Http {
+     public sealed class SocketsHttpHandler : HttpMessageHandler {
++        public QuicImplementationProvider QuicImplementationProvider { get; set; }
```

Comment:
@geoffkizer, @scalablecory, I thought we were going to rip this out before previews...?